Mural & tiny side quests

TheBoatyMcBoatFace opened the pull request:
Primarily added Mural documentation, but also updated the main README and added some stuff to .gitignore


<!-- readthedocs-preview civicactions-handbook start -->
----
📚 Documentation preview 📚: https://civicactions-handbook--1390.org.readthedocs.build/en/1390/

<!-- readthedocs-preview civicactions-handbook end -->

TheBoatyMcBoatFace committed:
Housekeeping

Make the URL prettier

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

TheBoatyMcBoatFace committed:
IT-249_Mural-Docs

1) Created a page for Mural in tools & support
2) Added a screenshot of where to find mural rooms
3) Added Mural to the dropdown menu

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

TheBoatyMcBoatFace committed:
Added Nova to Gitignore

Nova config is handled in the .nova directory. While this is clearly the best code editor out there, not everyone uses it. Until Nova takes over the MacWorld, I’m adding the .nova directory to the ignores

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

TheBoatyMcBoatFace committed:
Fixed list spacing

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

TheBoatyMcBoatFace committed:
Merge branch 'master' of https://github.com/CivicActions/guidebook into TheBoatyMcBoatFace-patch-1

TheBoatyMcBoatFace committed:
Unprettiering

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

dmundra commented on common-practices-tools/software-and-support/mural.md:
```diff
@@ -0,0 +1,59 @@
+  <a href="https://civicactions.slack.com/archives/C06PASS8S6M">
+    <img src="https://img.shields.io/badge/Slack_Channel-D83933?style=for-the-badge&logo=slack" alt="#mural Slack Channel">
+  </a><br><br>
+</div>
```
Can we use markdown here? We have ability to use https://squidfunk.github.io/mkdocs-material/reference/grids/ if needed.

dmundra replied:
Also capture the images as assets instead of linking to outside versions? That is minor though

dmundra commented on common-practices-tools/software-and-support/mural.md:
```diff
@@ -0,0 +1,59 @@
+
+## Enhancing Templates and Use Cases
+
+If your project has a specific use case or if you have ideas for improving our Mural templates, please share your suggestions in the [\#mural](https://civicactions.slack.com/archives/C06PASS8S6M) or [\#how-we-work](https://civicactions.slack.com/archives/C02HFM4KR) Slack channel by clicking the 🎫 emoji or submit a feature improvement through our [support portal](https://team.support.civicactions. ...
```
Minor issue here.

> Unexpected emphasis marker `_`, expected `*`

dmundra reviewed (changes requested):
@TheBoatyMcBoatFace the list indent issue should be fixed (see Files changed tab). I left you comments on some other changes.

TheBoatyMcBoatFace committed:
Fixing my passive-aggressive link formatting

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

TheBoatyMcBoatFace committed:
Merge branch 'TheBoatyMcBoatFace-patch-1' of https://github.com/CivicActions/guidebook into TheBoatyMcBoatFace-patch-1

TheBoatyMcBoatFace committed:
No Prettier

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

TheBoatyMcBoatFace committed:
Converted buttons to Markdown

Resolves: https://github.com/CivicActions/guidebook/pull/1390#discussion_r1638594330
Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

TheBoatyMcBoatFace committed:
Commented special characters

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

TheBoatyMcBoatFace committed:
Desnark

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

TheBoatyMcBoatFace committed:
Merge branch 'TheBoatyMcBoatFace-patch-1' of https://github.com/CivicActions/guidebook into TheBoatyMcBoatFace-patch-1

TheBoatyMcBoatFace committed:
Pre-commit will be the death of me

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

TheBoatyMcBoatFace committed:
Play nice Pre-commit & Slack

Pre-commit HATES Slack channels, specifically when you `\#` to show the `#` sign. I just renamed the buttons and links so they can play nice with pre-commit, my arch-nemesis.

Signed-off-by: TheBoatyMcBoatFace <bentley.hensel@civicactions.com>

dmundra reviewed (approved):
Thanks @TheBoatyMcBoatFace for fighting the good fight.

dmundra committed:
Merge branch 'master' into TheBoatyMcBoatFace-patch-1
